HiWARPs/cactus-backend
# Feature/file download

# What does this PR do? 

This PR adds the following endpoint: 

```
GET /results
```

This endpoint lets you download a CSV file containing results from a MongoDB query. 

Currently, it return a single row in the `electrons` collection.

# How to test this PR:

if you are connected to the DB and have data in the electrons collection, you can call the endpoint:

`curl -OJ -v http://localhost:3000/results`

You should get a file downloaded with the content being one of the rows in the electrons collections.

If not, start the app as usual:

$ make docker_build
$ export DATABASE_URL=mongodb://localhost:27017 && npm start
